requirements.txt

```diff
@@ -1,9 +1,26 @@
 asgiref==3.4.1
-dj-database-url==1.0.0
+black==22.6.0
+cfgv==3.3.1
+click==8.1.3
+distlib==0.3.5
+dj_database_url==1.0.0
 Django==3.2.14
 django-cors-headers==3.10.0
 django-extensions==3.1.5
 djangorestframework==3.12.4
+filelock==3.7.1
+identify==2.5.2
+isort==5.10.1
+mypy-extensions==0.4.3
+nodeenv==1.7.0
+pathspec==0.9.0
+platformdirs==2.5.2
+pre-commit==2.20.0
```

Comment:
I'm not a big fan of having this (and everything else related to linting) in the same place as the actuall app requirements... Maybe we can split in "requirements.txt" and "requirements-dev.txt"?

Reply:
That makes sense, although the tradeoff would be that users would have to install the `requirements-dev.txt` as well for local development (adding another step for them). Related, what do you think of packaging tools like [pip-tools](https://github.com/jazzband/pip-tools) or [Poetry](https://github.com/python-poetry/poetry)?